internal/controllers/reconcile_test.go

```diff
@@ -1275,10 +1275,10 @@ service:
 	require.NoError(t, clientErr)
 	// wait until the reconciler sees the object as deleted in its cache
 	assert.EventuallyWithT(t, func(collect *assert.CollectT) {
-		actual := &v1beta1.OpenTelemetryCollector{}
+		actual := &v1alpha1.OpenTelemetryCollector{}
```

Comment:
Why are we changing this?

Reply:
The test further up defines it as v1alpha1. I can also change both to beta.